docs/core/responses.rst

```diff
@@ -38,6 +39,34 @@ you need to retrain the assistant before these changes will be picked up.
 More details about the format of these responses can be found in the
 documentation about the domain file format: :ref:`utter_templates`.
 
+.. _direct-response:
+
+Managing responses to direct response intents
+----------------------------------------------
+
+All assistant utterances which should be used as a reply to direct response intents can be added to a separate NLG file
+as part of the NLU training data. The contents of
+
+For example, if you specified a direct response intent as -
+
+.. code-block:: md
+
+    ## intent: faq/python_version
+    - which python version should i install
+    - what version of python
+    - which python do you support?
+
+the NLG file with the assistant's utterance for the corresponding intent would look like -
+
+.. code-block:: md
+
+    * faq/python_version
```

Comment:
maybe choose a different, more common example. (also, this would show up in docs search results to possible confusion of users)